.travis/check-custom-style

```diff
@@ -85,6 +86,23 @@ function avoidLegacyApi() {
   displayResult "$found"
 }
 
+function removeUnusedLogAnnotations() {
+  title "Remove unused lombok @Log and @Slf4j annotations"
+  local found=0
+
+  while read -r file; do
+    if grep -Eq "@Log|@Slf4j" "$file"; then
+      if ! grep -Eq "log::|log\." "$file"; then
+        echo -e "${bold}$file${normal}"
+        found=1
+        status=1
+      fi
+    fi
+  done <<< "$(cat "$javaFiles")"
+
+  displayResult "$found"
```

Comment:
Side-Note:
I see that you're just flipping this between 1 and 0, did you consider using this as a counter similar to checkstyle to say "Found 1337" issues at the end?